.github/ISSUE_TEMPLATE/developer-onboarding.md

```diff
@@ -24,11 +24,12 @@ There are several tools we use locally that you will need to have.
 - [ ] [Create a cloud.gov account](https://cloud.gov/docs/getting-started/accounts/)
 - [ ] Have an admin add you to the CISA Github organization and Dotgov Team.
 - [ ] Ensure you can login to your cloud.gov account via the CLI
-```
+```bash
 cf login -a api.fr.cloud.gov  --sso
 ```
 - [ ] Have an admin add you to cloud.gov org and relevant spaces as a SpaceDeveloper
-```cf set-space-role cloud.account@email.gov ORG SPACE SpaceDeveloper
+```bash
+cf set-space-role cloud.account@email.gov ORG SPACE SpaceDeveloper
```

Comment:
Question, do we know any of the placeholder information for ORG and SPACE already that we could put in here? Or will it be changing? 

Also wonder if it's worth putting cloud.acount@email.gov in brackets (i.e. `<cloud.account@email.gov>` ) to emphasize it is a placeholder

Reply:
We do but it will become stale soon when we move to a new org. I'll add for now and we can update later!